src/pools/ID3MPool.sol

```diff
@@ -26,18 +26,6 @@ pragma solidity ^0.8.14;
     guidelines for assets/shares/maxWithdraw etc.
 */
 interface ID3MPool {
-    /**
```

Comment:
Consider changing the sentence `msg.sender must be authorized.` in `depost`, `withdraw` and `transfer` to `msg.sender must be the hub`.